packages/tenable_io/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.1"
```

Bump up the minor version.
```suggestion
- version: "0.2.0"
```

---

Hey @kcreddy,

Looking into other PR's, this change is relatively very tiny which was requested by user. So that's why we increased the patch version.

---

While we don't have documentation around it, I think it has to be bumped in minor version as UI gets changed from user perspective.

---

We will bump up the minor version.